packages/vscode-mdx/script/build.mjs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+import {createRequire} from 'node:module'
+import process from 'node:process'
+import {fileURLToPath} from 'node:url'
+import {build} from 'esbuild'
+
+const require = createRequire(import.meta.url)
+
+const debug = process.argv.includes('debug')
+
+await build({
+  bundle: true,
+  entryPoints: {
+    extension: require.resolve('../src/extension.js'),
+    'language-server': require.resolve('@mdx-js/language-server')
+  },
+  external: ['vscode'],
+  logLevel: 'info',
+  minify: !debug,
+  outdir: fileURLToPath(new URL('../out/', import.meta.url)),
+  platform: 'node',
+  sourcemap: debug,
```

Comment:
Wouldn't/shouldn't source maps be enabled when minification is done?

Reply:
I kept the same parameters as the original script (which I wrote).

Currently that would 16.6MB to the size of the extension, which is a lot compared to the 4MB size of the extension without source maps. On the other hand, it’s just a file on disk, not on a website or anything.

I’m not really for nor against adding the source map in production.